r_markdown/carbonyls.Rmd

```diff
@@ -123,7 +123,8 @@ Load notes
                                     -id, -id_ions, -flow,
                                     -start, -end, -type, -qc) %>%
                       dplyr::filter(!is.na(pol)) %>%
-                      dplyr::mutate(pol = as.factor(pol))
+                      dplyr::mutate(pol = as.factor(pol)) %>%
+                      dplyr::select(-id_ions)
```

Comment:
- umm, id has to be called id or chaos will ensue???

- what matters with WSOC is which sampler it comes from otherwise flow rates won't match up... What is?